src/js/components/Table/StyledTable.js

```diff
@@ -60,9 +59,7 @@ const StyledTableDataCaption = styled.caption`
 StyledTableDataCaption.defaultProps = {};
 Object.setPrototypeOf(StyledTableDataCaption.defaultProps, defaultProps);
 
-const StyledTableRow = styled.tr`
-  height: 100%;
-`;
+const StyledTableRow = styled.tr``;
```

Comment:
Was the height safe for removal?

Reply:
I believe so. I've run through a bunch of scenarios to validate and haven't uncovered any issues yet.